### DISCUSSION TITLE
Possible standalone Linux kernel install from ingenic

### DISCUSSION THREAD

**Discussion by rickgitdone** [gtxaspec/wz_mini_hacks#87] [General]
http://www.ingenic.com.cn/en/?newsshow/tp/228/id/62.html   ...not sure how new the source is however it may shine a light on how to get past the iCamera lock on these Cams

*** page snip ***
Previously Summary

Because Linux and Android's share of a kernel code, we use Android source code download environment to download Linux kernel source.

The old download command: git clone http://git.ingenic.cn/android/kernel/kernel-3.0.8.git-bing-jz4780;

Uboot source is provided only by compressed package and uboot have no version control.

In early June 2013, Engineering Center Software Division began preparing Linux Git repository. After two weeks of the summary, discussion, upload, correction,synchronization, Git repository is complete on June 19, 2013.

 

Download repo

    $ mkdir 

    $ cd 

    $ wget http://git.ingenic.cn:8082/bj/repo

    $ chmod +x repo

 

Download source

    $./repo init -u http://git.ingenic.cn/linux/manifest -m outside.xml

    $./repo sync

 

Support and Feedback

 Feel free to contact support@ingenic.cn

**Comment by rickgitdone** [comment]
found a better candidate that seems to be active...

https://openipc.github.io/firmware/

T31 status is MVP ... nearly 100% working .. however I am too green to try this implementation...

**Comment by gtxaspec** [comment]
the problem with openipc, it's not completely open source, `magestic` which is their rtsp/http/everything server is only distributed as a precompiled binary... meaning you are reliant on them for anything besides the kernel.  So probably a no go there.

**Comment by rickgitdone** [comment]
found another asset for T31 Ingenic .... SDK possible from ISSIL 

https://www.lumissil.com/assets/pdf/core/T31-SNIPE_PB.pdf

**Reply by LouDnl** [reply]
Zeratul SDK is not available anywhere...

**Reply by DevopsDmytro** [reply]
Is It not available yet?
